db/seeds.rb

```diff
@@ -31,7 +34,22 @@ def create_stickers
                                       image_url: "http://devstickers.com/assets/img/cat/chrome.png",
                                       price: 2,
                                       description: "Chrome logo")
-    puts "#{Sticker.all.map(&:title).join(", ")} created."
+    puts "#{Sticker.all.map(&:title).join(", ")} stickers created."
+  end
+
+  def create_users
+    @sekhar = User.create(first_name: "Sekhar", last_name: "Paladugu", username: "sekhar", password: "sekhar")
+    @emily = User.create(first_name: "Emily", last_name: "Dowdle", username: "emily", password: "emily")
+    @jason = User.create(first_name: "Jason", last_name: "Pilz", username: "jason", password: "jason")
+    puts "#{User.all.map(&:username).join(", ")} users created."
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Do not write to stdout. Use Rails' logger if you want to log.